machine/v1beta1/types_awsplacementgroup.go

```diff
@@ -0,0 +1,164 @@
+package v1beta1
```

Comment:
Since we support upgrades on this and we know we'll never want to deal with migration of customer manifests, I think this is really v1.